app/views/ansible_roles/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% title _("Ansible Roles") %>
```

Comment:
This is missing a 'Welcome' page following Patternfly's empty state pattern:
https://www.patternfly.org/pattern-library/communication/empty-state/#_

You can pretty much copy the environments page from core, changing the icon to fa-play:
https://github.com/theforeman/foreman/blob/develop/app/views/environments/welcome.html.erb